src/components/popup/NewProductAdmin.js

```diff
@@ -189,7 +206,7 @@ export default class NewProductAdmin extends React.Component {
                   <Label for="quantity" sm={4}>Skladem</Label>
                   <Col sm={8}>
                     <InputGroup>
-                      <Input type="number" name="quantity" id="quantity" defaultValue="0"/>
+                      <Input onChange={this.onInputChange} type="number" name="quantity" id="quantity" defaultValue="0"/>
```

Comment:
místo defaultValue, dej value={this.state.quantity}, a tu defaultní hodnotu 0 dej nahoru do initial hodnoty statu